README.md

```diff
@@ -1,2 +1,53 @@
-# Capstone-Module-2
-Web application based on an external API. The API provides data about a topic.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Module 2
+
+
+> Web application that get data from https://www.themealdb.com/api.php maintains likes, comments and reservations using Involvement API.
+
+
+## Built With
+
+- Technologies used: HTML, CSS, Javascript, Webpack, Jest
+
+## Live Demo (if available)
+
+[Live Demo Link](https://neckerfree.github.io/Capstone-Module-2/)
+
+[Presentation Video](https://www.loom.com/share/51f6bd1fdc254014817f131c54404393)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+- Click the green `Code` button on the repo
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/NeckerFree/Capstone-Module-2`
+- On the terminal, navigate into the directory like so: `cd Capstone-Module-2`
+
+### Install
+- in visual studio code build with the command `npm run build` 
+- and run the distribution with the command `npm start`
+
+## Authors
+
+👤 **Elio Cortés**
+
+- GitHub: [@NeckerFree](https://github.com/NeckerFree)
+- Twitter: [@ElioCortesM](https://twitter.com/ElioCortesM)
+- LinkedIn: [elionelsoncortes](https://www.linkedin.com/in/elionelsoncortes/)
```

Comment:
- [ ] **Required:** Kindly add all contributors to the project. This makes your documentation more professional.

Reply:
I had some difficulties with my partner, and I was unpaired: That’s why I’m presenting this project individually.

Reply:
This is clear 👍🏽